src/renderer/utils/store.ts

```diff
@@ -151,14 +151,6 @@ export function setPaginatedResultErrorReducer() {
   };
 }
 
-export function unsetActiveNodeReducer() {
```

Comment:
maybe we will need this method in the future

Reply:
We can add it in back in the future if we do need it. But if we are unsetting an active node to set another node, we can use the `changeActiveNodeReducer`